daprd/Dockerfile

```diff
@@ -1,2 +1,5 @@
-FROM daprio/daprd:1.9.6
-COPY --from=busybox:1.28 /bin/sh /bin/sh
\ No newline at end of file
+FROM daprio/daprd:1.11.0-rc.5 as builder
```

Comment:
@mcruzdev I don't think we need this container anymore.. we just need to point to that image in the helm templates using a value right? 
The main reason to consume the bundles from a configMap is to not need a custom version of `daprd`